content/blog/pulumi-service-improvements_02-2020/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: "Pulumi Console Improvements, February 2020"
+h1: "Pulumi Console Improvements, February 2020"
+authors: ["chris-smith"]
+tags: ["pulumi","new-features"]
+date: "2020-02-28"
+meta_desc: "Recent improvements to the Pulumi Console: stack tags, audit logs, CI/CD integration, downloadable checkpoint files"
+meta_image: "pretty-print-multiline-json.gif"
+---
+
+Hello! Many posts here on the [Pulumi blog]({{< relref "/blog" >}}) describe new features in our core libraries or
+command-line client. But in this post, we'll highlight some of the recent improvements to the [Pulumi Console](https://app.pulumi.com).
+
+We've been hard at work making it easier to manage stacks, permissions, and organizations. So we're super-excited to showcase
+what's new!
+
+<!--more-->
+
+## First-class Support for Tags
+
+Pulumi has had support for [stack tags]({{< relref "/docs/intro/concepts/stack#stack-tags" >}}) for a while, enabling
+you to add attribute your stacks with custom data such as the `account-id` or `environment`. But previously the data was
+only available on the command-line, via the `pulumi stack tag` command ([documentation]({{< relref "/docs/reference/cli/pulumi_stack_tag" >}})).
+
+We've added first-class support for stack tags. You can now create, update, and delete tags from within the Pulumi Console.
+
+![Updating stack tags within the Pulumi Console](./update-tags-in-console.png)
+
+But making it easier to create and edit tag data is just the beginning. We've started to add new search/filtering capabilities
+based on stack tags. For example, you can now use filters to more quickly add stacks to [Pulumi teams]({{< relref "/docs/intro/console/collaboration/teams" >}}).
+
+![Filtering stacks by their tags](./filter-stacks-by-tag.png)
+
+We will continue wiring stack tags throughout other parts of the Pulumi Console, so stay tuned! Also, this new feature was
+added by Pulumi's first (and extremely awesome) inter, Tasia (👋)!
```

Comment:
```suggestion
added by Pulumi's first (and extremely awesome) intern, Tasia (👋)!
```

Perhaps also a link to Tasia's GitHub?

Reply:
I didn't want to draw too much attention to it in case @tasiah didn't want TO BECOME A CELEBRITY. Would you feel comfortable with us linking to your GitHub profile Tasia? (Or any other URL, like a personal blog or something?)

Reply:
Feel free to add my Github or LinkedIn. I don't mind either way :)